roles/ocp4_workload_keycloak/templates/keycloak_instance.yml.j2

```diff
@@ -0,0 +1,30 @@
+apiVersion: k8s.keycloak.org/v2alpha1
+kind: Keycloak
+metadata:
+  name: keycloak
```

Comment:
Namespace?

Reply:
I've set that in the workload.yaml in the k8s module